### PR TITLE
Semver updates based on WG LC

### PR DESCRIPTION
**Pull request by xorrkaz**

### CODE REVIEW

**Comment by xorrkaz** [comment]
Comments brought up on the last call have been addressed.

**Comment by Reshad-Rahman** [comment]
Took a look, changes look good. Nits etc can wait IMO.

File: yang-semver/draft-ietf-netmod-yang-semver.xml [hunk 1]
```diff
@@ -191,24 +199,28 @@
 <li>'Y' is the MINOR version.  Changes in the MINOR version number
        indicate changes that are backwards-compatible to versions with
        the same MAJOR version number, but a lower MINOR version number
-       and no PATCH "_compatible" or "_non_compatible" modifier.</li>
+       and no "_compatible" or "_non_compatible" modifier.</li>
+<li>'Z' is the PATCH version. Changes in the PATCH
+       version number can indicate an editorial change to the YANG artifact.
+       In conjunction with the '_COMPAT' modifier (see below) changes to 'Z'
+       may indicate a larger module change.
```
Comment (jsterne) [inline yang-semver/draft-ietf-netmod-yang-semver.xml:206]: not a huge fan of "a larger module change". We ended up using the term "impactful" in my recent Module Versioning updates, so perhaps "may indicate a more impactful module change" ?

Comment (xorrkaz) [reply yang-semver/draft-ietf-netmod-yang-semver.xml:206]: I'm not a fan, either.  But I don't like the word "impactful."  What about, "a more substantive module change"?

File: yang-semver/draft-ietf-netmod-yang-semver.xml [hunk 2]
```diff
@@ -871,11 +891,17 @@ module ietf-yang-semver {
...
+      T1: 1.1.0-draft-jdoe-netmod-exmod-enhancements-00
+        |
+      T1: 1.1.0-draft-jdoe-netmod-exmod-enhancements-01
+]]></artwork>
+<t keepWithNext="true">In parallel with (track 2):</t>
+<artwork name="" type="" align="left" alt=""><![CDATA[
+      T2: 1.1.0-draft-jadoe-netmod-exmod-changes-00
+        |
+      T2: 1.1.0-draft-jadoe-netmod-exmod-changes-01
 ]]></artwork>
```
Comment (jsterne) [inline yang-semver/draft-ietf-netmod-yang-semver.xml:905]: I think maybe we got the T1/T2 slightly wrong here ?  Shouldn't the first two lines be T1 then T2 (for jdoe) and then similarly T1 then T2 for jadoe ?

Also - the difference between jdoe and jadoe is too subtle. How about jdoe and some totally different name like "fredsmith" ?

File: yang-semver/draft-ietf-netmod-yang-semver.xml (hunk 1 above)
Comment (jsterne) [reply yang-semver/draft-ietf-netmod-yang-semver.xml:206]: I'm not loving that either, but don't care too much so if you're keen on substantive then it is probably clear enough.

File: yang-semver/draft-ietf-netmod-yang-semver.xml (hunk 2 above)
Comment (jsterne) [reply yang-semver/draft-ietf-netmod-yang-semver.xml:905]: When I suggested T0, T1 I was thinking *time*. I see you were thinking "Track".  They are already sorted by track (with your new layout) so I'd vote for time, but it seems we should explain what T0, T1 are :-)

File: yang-semver/draft-ietf-netmod-yang-semver.xml (hunk 1 above)
Comment (xorrkaz) [reply yang-semver/draft-ietf-netmod-yang-semver.xml:206]: Okay.  I'll make the change.

File: yang-semver/draft-ietf-netmod-yang-semver.xml (hunk 2 above)
Comment (xorrkaz) [reply yang-semver/draft-ietf-netmod-yang-semver.xml:905]: I think time is already conveyed (i.e., one rev after the other).  It was the move from side-by-side to stacked that I didn't know conveyed track well enough.  I get your point that Tn implies time in a lot of cases.  I think I'll yank it out and change the name to be clearer.

**Comment by xorrkaz** [comment]
@jsterne Okay to merge this and start work on branching text?

**Comment by jsterne** [comment]
Yeah - let's go ahead with these changes and work on the branching text for another update.

**Pull request merged**